engine/apps/email/inbound.py

```diff
@@ -69,7 +69,7 @@ def post(self, request, alert_receive_channel):
                 alert_receive_channel_pk=alert_receive_channel.pk,
                 image_url=None,
                 link_to_upstream_details=None,
-                integration_unique_data=request.data,
+                integration_unique_data=None,
```

Comment:
This data is not needed, the only fields available for templating are subject, message, and sender.